appveyor.yml

```diff
@@ -0,0 +1,23 @@
+# Inspired by appveyor in xarray and anaconda-project
+
+build: false
+
+environment:
+  global:
+    CONDA: "C:\\Miniconda36-x64"
+
+install:
+  - "SET PATH=%CONDA%;%CONDA%\\Scripts;%PATH%"
+  # The safety checks are simply intended to ensure that there is enough disk space
+  # and the user has the necessary permissions to make environment changes. In a CI
+  # environment these are not necessary and slow things down noticeably on Windows.
+  - conda config --set always_yes yes --set changeps1 no --set auto_update_conda no --set safety_checks disabled --add channels conda-forge
+  - conda update -q conda
+  - conda env create -n test_env python=3.6 pytest
+  - conda activate test_env
+  - conda install --file requirements.txt
```

Comment:
normally requirements.txt should be the pip names of the packages, and conda names may differ. There may also be additional, optional packages that we want for the purpose of testing.
cc @bryevdv

Reply:
Think I can just take all that directly? I wasn't sure that we need to build intake to just run the tests

Reply:
Probably so? It's the window analogue of this test install step https://github.com/ContinuumIO/intake/blob/master/scripts/ci/install.sh#L26-L27

Reply:
ok I switched it